README.md

```diff
@@ -269,6 +269,14 @@ Execute a sync but only log API calls to Jira which would create/modify Jira Iss
 
 **Description:** Comma separated list of User Emails for Atlassion Jira or User EUA IDs for Enterprise Jira.
 
+### `jira-watchers`
+
+**Required: No**
+
+**Default Value: ''**
+
+**Description:** Comma separated list of Labels to be added on newly created issues.
+
 ## Local Testing
```

Comment:
I take it this was missed from the last PR and this is only adding the documentation for jira-watchers?